openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -62,17 +65,17 @@
 log = logging.getLogger("edx.student")
 AUDIT_LOG = logging.getLogger("audit")
 USER_MODEL = get_user_model()
-PASSWORD_RESET_INITIATED = 'edx.user.passwordreset.initiated'
```

Comment:
I apologize for mixing reformatter auto fixes with  actual changes in a single PR. Assume everything in this file except the bit that I am going to flag with a comment below is just automatic fixes  to make the linter happier.